package-sets/1.0.0.json

```
@@ -41,6 +41,24 @@
     },
     "either": "6.1.0",
     "exists": "6.0.0",
+    "foldable-traversable": {
+      "git": "https://github.com/purescm/purescript-foldable-traversable.git",
+      "ref": "9b265472a9e8a2cbcf21a756b0d7e584eb2c6fdf",
+      "dependencies": [
+        "prelude",
+        "bifunctors",
+        "const",
+        "control",
+        "either",
+        "functors",
+        "identity",
+        "maybe",
+        "newtype",
+        "orders",
+        "prelude",
+        "tuples"
+      ]
```

You do not need to list the dependencies if the upstream has a `spago.yaml` file

```suggestion
      "ref": "9b265472a9e8a2cbcf21a756b0d7e584eb2c6fdf"
```

---

Ah I see, I was wondering whether I needed those. I removed a bunch of those now.

---

Nice! 🙂